### ISSUE TITLE
Phalcon 4

### ISSUE THREAD

**Issue by ruudboon** [CpanelInc/ea-scl-phalcon#6]
Would it be possible to add a new repo for Phalcon V4? The v4 beta is out now.

**Comment by gitcory** [comment]
I'll look into it, but if we do it would have to be in the experimental repo until it comes out of Beta. I do already have a case pending its release from Beta to put into production.

**Comment by ruudboon** [comment]
Thnx. I think it would be good if both (3&4) exists next to each other for a while.

**Comment by gitcory** [comment]
Agreed, as long as you don't mind using the EA4-experimental repo I don't see a reason not get to get a head start on phalcon4

**Comment by ruudboon** [comment]
Yeah off-course. EA4-experimental repo sounds good to me.

**Comment by ismail0234** [comment]
@ruudboon  Will v3 version be removed when v4 is released?

**Comment by drmuey** [comment]
@ismail0234 @ruudboon Probably they'd be two different packages that conflicted with each other
